src/payment.js

```diff
@@ -179,6 +180,26 @@ export class Payment {
             redirect_uri: redirectUri
         });
     }
+
+    /**
+     * Get the url for flow to purchase a promo code product with ZUORA
+     * @param {string} code - promocode product code
+     * @param {string} [state=''] - An opaque value used by the client to maintain state between
+     * the request and callback. It's also recommended to prevent CSRF
+     * @param {string} [redirectUri=this.redirectUri]
+     * @return {string} - The url to the buy promo code product flow
+     */
+    purchasePromoCodeProductFlowUrl(code, state = '', redirectUri = this.redirectUri) {
```

Comment:
I'd drop `Flow` part from name

Reply:
Looks like other methods have `Flow` in names `purchaseProductFlowUrl`, `purchaseCampaignFlowUrl`